config/application.rb

```diff
@@ -36,5 +36,6 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.autoloader = :classic
```

Comment:
Why?

Reply:
@jandudulski I had problem that there was no loading concepts modules and classes. I typed in duckduckgo and find this way to autoload.

Reply:
When you find some time try to adjust to the newer autoloader ;)